src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {` ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          I am
+          {` ${age}`}
+        </p>
+      )}
+
+      {
+        (isMarried && (
+          <p className="Person__partner">
+            {
+              sex === 'm'
+                ? `${partnerName} is my wife`
+                : `${partnerName} is my husband`
+            }
+          </p>
+        ))
+        || <p className="Person__partner">I am not married</p>
```

Comment:
It's better to use ternary operator instead of && with ||
As it will make your code cleaner